.github/workflows/release.yml

```
@@ -0,0 +1,124 @@
+name: Release
+
+permissions:
+  contents: write
+
+on:
+  push:
+    tags:
+      - v[0-9]+.*
+
+jobs:
+  # First we are going to create a task that generates a new release in GitHub
+  # as a draft. All the wheels will end up being uploaded here at the end.
```

```suggestion
  # as a draft. All the wheels will end up being uploaded here when complete.
```

`end` was being used twice in this sentence.